rac-gcp-deploy/orb.yml

```diff
@@ -467,62 +489,3 @@ jobs:
       - snyk_monitor
       - persist_workspace:
           to: <<parameters.workspace-dir>>
-
-  prd-deploy:
-    parameters:
-      executor:
-        description: "Name of executor to use for this job. Defaults to docker executor"
-        type: executor
-        default: docker
-      workspace-dir:
-        description: "Path to restore the workspace"
-        type: string
-        default: ~/project
-      google-compute-zone:
-        description: "The Google compute zone to connect with via the gcloud CLI. Defaults to europe-west1-b"
-        type: string
-        default: europe-west1-b
-      cluster-name:
-        description: "Name of the environment variable storing the Kubernetes cluster name"
-        type: env_var_name
-        default: K8S_CLUSTER_NAME
-      helm-release-name:
-        description: "Name of environment variable storing the helm release name"
-        type: env_var_name
-        default: CIRCLE_PROJECT_REPONAME
-      prd-gcloud-service-key:
-        description: "Name of environment variable storing the full service key JSON file for the PRD Google project"
-        type: env_var_name
-        default: PRD_GCLOUD_ACCOUNT_AUTH
-      prd-project-id:
-        description: "Name of environment variable storing the PRD Google project ID to connect with via the gcloud CLI"
-        type: env_var_name
-        default: PRD_PROJECT_ID
-      container-name:
-        description: "Name of environment variable storing the name of the container we are publishing"
-        type: env_var_name
-        default: CIRCLE_PROJECT_REPONAME
-      registry-url:
-        description: "The GCR registry URL. Defaults to eu.gcr.io"
-        default: eu.gcr.io
-        type: string
-      cache-suffix:
-        description: "Suffix to use for the cache key, e.g. -v2, to be able to clear the cache"
-        default: ""
-        type: string
-    executor: <<parameters.executor>>
-    steps:
-      - restore_workspace:
-          to: <<parameters.workspace-dir>>
-      - load_cache:
-          cache-suffix: <<parameters.cache-suffix>>
-      - gcloud_auth: *gcloud_auth_prd
-      - k8s_auth:
```

Comment:
can this command be removed now? seems like it's not used anymore

Reply:
Yes!!!